site/scripts/build_entry.sh

```diff
@@ -6,6 +6,12 @@ curl https://umami.quacs.org/umami.js > public/umami.js || exit 1
 # If umami crashes, the file will contain an nginx error instead of the js code
 if cat public/umami.js | grep -q nginx; then exit 1; fi
 
+echo Getting time of last data scraped
+curl "https://api.github.com/repos/quacs/quacs/actions/runs?per_page=100&event=schedule" > temp.json
+jq 'first(.workflow_runs[] | select(.conclusion=="success")) | .updated_at' temp.json > src/store/data/latest_scrape.json
+printf '%s%s%s' "{\"last_updated\":" "$(cat src/store/data/latest_scrape.json)" "}" > src/store/data/latest_scrape.json
+rm temp.json
```

Comment:
I don't see why this file is necessary when the `meta.json` file already has the time of scraping.  Wouldn't it be easier to just pull the relevant times from that file (especially since they're known to be tied to the state of the git repository which is being used to build the site)?

Reply:
To piggyback on this - you could simply use the current build time if you're worried about the string not updating when we push a commit that affects the site outside of scraping.